packages/lesswrong/components/dialogues/DialogueRecommendationRow.tsx

```diff
@@ -0,0 +1,230 @@
+// TODO: Import component in components.ts
+import React, {useState} from 'react';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import { useTracking } from "../../lib/analyticsEvents";
+import {useCurrentUser} from '../common/withUser';
+import {gql, useQuery} from '@apollo/client';
+import {DialogueUserRowProps} from '../users/DialogueMatchingPage';
+import classNames from 'classnames';
+
+const styles = (theme: ThemeType) => ({
+  dialogueUserRow: { // TODO; shared import
```

Comment:
What is this TODO referring to?

Reply:
Wondering if I should be sharing classes across files instead of having repeated code. Probably should

Reply:
and I guess the way to do that is to pass a classes argument into the component

Reply:
actually, leaving this for now, since am making changes in the child file to the styles such that the added complexity of DRY doesn't seem worth it